explore-analyze/discover/background-search.md

```diff
@@ -25,11 +25,31 @@ If you have been using search sessions and upgrade to 9.2, your search sessions
 
 ## Enable background searches
 
-This feature is disabled by default. You can enable background searches in versions 9.2 and later, or search sessions in versions 9.1 and earlier, by setting [`data.search.sessions.enabled`](kibana://reference/configuration-reference/search-sessions-settings.md) to `true` in the [`kibana.yml`](/deploy-manage/stack-settings.md) configuration file.
+:::::{applies-switch}
+
+::::{applies-item} serverless: ga
+This feature is already enabled by default.
+::::
+
+::::{applies-item} stack: ga 9.3
+This feature is already enabled by default.
```

Comment:
nit but you could tighten this to: `This feature is enabled by default.`

I think that's typically how we phrase it in the docs.